toolchain/toolchain.bzl

```diff
@@ -35,6 +35,10 @@ target_constraints = {
         "arm": ["@platforms//os:linux", "@platforms//cpu:arm"],
         "armv7": ["@platforms//os:linux", "@platforms//cpu:armv7"],
     },
+    "aarch64-none-linux-gnu": {
+        "arm64": ["@platforms//os:linux", "@platforms//cpu:arm64"],
+        "aarch64": ["@platforms//os:linux", "@platforms//cpu:aarch64"],
```

Comment:
nit: I think that [`arm64` is just an alias of `aarch64`](https://github.com/bazelbuild/platforms/blob/2af915c086d82ab0b233c8ff420ca72e128b8790/cpu/BUILD#L97-L100) so we can probably just use one here.

```starlark
alias(
    name = "arm64",
    actual = ":aarch64",
)
```

This is not blocking